imgparse/_version.py

```diff
@@ -1,3 +1,3 @@
 """Defines package version.  Parsed by setup.py and imported by __init__.py."""
 
-__version__ = "1.13.0"
+__version__ = "1.13.1"
```

Comment:
Oh also this is enough changes that it is definitely worthy of a minor version bump

Reply:
https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/128f96f0bb5a2d923c00bdf2fdef61f780c3febb Changed to 1.14.0